.babelrc

```diff
@@ -1,14 +1,21 @@
 {
     "presets": [
         [
+
             "@babel/preset-env",
             {
                 "targets": {
                     "node": "current"
                 }
             }
         ],
-        "@babel/preset-react"
+        "@babel/preset-react",
+        [
+            "react-app",
+            {
+                "absoluteRuntime": false
+            }
+        ]
```

Comment:
react-app line causes error when transpiling

Error: [BABEL] /Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@exabyte-io/materials-designer/src/MaterialsDesigner.js: Using `babel-preset-react-app` requires that you specify `NODE_ENV` or `BABEL_ENV` environment variables. Valid values are "development", "test", and "production". Instead, received: undefined. (While processing: "/Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/babel-preset-react-app/index.js")
    at module.exports (/Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/babel-preset-react-app/create.js:58:11)
    at module.exports (/Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/babel-preset-react-app/index.js:19:10)
    at /Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@babel/core/lib/config/full.js:165:14
    at cachedFunction (/Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@babel/core/lib/config/caching.js:33:19)
    at loadPresetDescriptor (/Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@babel/core/lib/config/full.js:235:63)
    at /Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@babel/core/lib/config/full.js:77:21
    at Array.reduce (<anonymous>)
    at recurseDescriptors (/Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@babel/core/lib/config/full.js:74:38)
    at loadFullConfig (/Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@babel/core/lib/config/full.js:108:6)
    at /Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@babel/core/lib/transform-file.js:48:33
Rendering Complete, saving .css file...
Wrote CSS to /Users/jhanson/exabyte-stack/web-app/src/exabyte/node_modules/@exabyte-io/materials-designer/dist/stylesheets/main.css